src/experimental/safe_html_transform/testing.py

```diff
@@ -23,6 +25,8 @@ def setUpZope(self, app, configurationContext):
             experimental.safe_html_transform,
             context=configurationContext
         )
+        logger.root.setLevel(logging.DEBUG)
+        logger.root.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
Don't leave logging to stdout on all the time. Either remove this or make it conditional somehow.